src/image/utils/schema/triggers.py

```diff
@@ -28,10 +28,9 @@ class Config:
         extra = pydantic.Extra.forbid
 
     @pydantic.validator("end_of_life")
-    @classmethod
-    def ensure_still_supported(cls, v: datetime) -> datetime:
+    def ensure_still_supported(cls, v: Optional[datetime]) -> Optional[datetime]:
         """ensure that the end of life isn't reached."""
-        if v < datetime.now(timezone.utc):
+        if v is None or v < datetime.now(timezone.utc):
```

Comment:
this is a good point. since it's simple, can we force this field to always be a date? (so, no default)

now that you are introducing this verification mechanism, there are no excuses to not provide eol

Reply:
in that case, you probably want it to be not Optional anymore. Is there a valid use-case where eol is not required?

Reply:
@cjdcordeiro we will need to update mock-rock's release/test to include an end-of-life in that case.

Reply:
sounds good

Reply:
@cjdcordeiro All done.